tutorials/02-example.py

```diff
@@ -7,6 +7,450 @@
+#
+# We are going to use the command ``mapdl.vdrag`` to create a volume by dragging
+# an area along a line.
+help(mapdl.vdrag)
+
+#################################################################################
+# First, let's define the lince we are going to use to drag the area along.
+lenght_wing = 1.5  # [m] MAPDL is unit agnostic.
+
+k0 = mapdl.k("", 0, 0, 0)
+kz = mapdl.k("", 0, 0, lenght_wing)
+
+ldrag = mapdl.l(k0, kz)
+
+#################################################################################
+# Create the volume
+vol0 = mapdl.vdrag("all", nlp1=ldrag)
+
+#################################################################################
+# Let's check the results
+mapdl.vplot()
+
+
+#################################################################################
+# Finite element definition
+# =========================
+#
+# As you all know, finite element approaches split the domains into "finite elements"
+# where you solve your equations in their quadrature points.
+# Therefore, we need to define how that domain split is going to be perform, aka
+# "choosing element type".
+#
+
+# Defining element type
+mapdl.et(1, "SOLID187")
+
+##################################################################################
+# Here we could also define other element options using the ``mapdl.keyopt``
+# command.
+
+
+#################################################################################
+# Material definition
+# ===================
+#
+# Let's define the material our wing is made off. We are going to choose steel
+# in its simplest configuration (elastic linear material). But you could define
+# other parameters (plasticity, fatige, viscoelasticity, etc).
+#
+
+# Define a material (nominal steel in SI)
+mapdl.mp("EX", 1, 210e9)  # Elastic moduli in Pa (kg/(m*s**2))
+mapdl.mp("DENS", 1, 7800)  # Density in kg/m3
+mapdl.mp("NUXY", 1, 0.3)  # Poisson's Ratio
+
+#################################################################################
+# Mesh Generation
+# ===============
+#
+# Let's finally split the domain:
+maximum_element_size = 1 / 20
+mapdl.esize(maximum_element_size)
+
+mapdl.vmesh("all")  # Mesh
+
+#################################################################################
+# Let's check the results:
+mapdl.eplot()
+
+
+#################################################################################
+# Boundary Conditions Definition
+# ==============================
+#
+# Let's fix the nodes at the origin to not move.
+
+mapdl.nsel("s", "loc", "z", 0)
+mapdl.d("all", "all", 0)
+
+##################################################################################
+# Wind Excitation
+# ---------------
+#
+# Let's apply a excitation to our wing. However, we don't really know what win
+# speed to apply, so let's pull some online data first.
+#
+# We are going to retrieve some data from NASA regarding the wind speed at
+# Ansys Madrid office in Paseo de la castellana.
+#
+# .. figure:: ../images/office1.jpg
```

Comment:
Smooth way of saying, look at our office! its nice! haha

Reply:
hahahaha... we need to get some interns/hires!! hahahah